OloEditor/src/Panels/TerrainEditorPanel.cpp

```
@@ -657,6 +665,41 @@ namespace OloEngine
         }
     }
 
+    void TerrainEditorPanel::OnVoxelUpdate(Ref<VoxelOverride> voxels, const VoxelRayHit& hit, bool mouseDown)
```

Cppcheck (2.21.0)

[style] The function 'OnVoxelUpdate' is never used.

(unusedFunction)